x/blocksdk/keeper/keeper.go

```diff
@@ -0,0 +1,82 @@
+package keeper
+
+import (
+	"fmt"
+
+	"cosmossdk.io/log"
+	"cosmossdk.io/store/prefix"
+	storetypes "cosmossdk.io/store/types"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/skip-mev/block-sdk/x/blocksdk/types"
+)
+
+type Keeper struct {
+	cdc      codec.BinaryCodec
+	storeKey storetypes.StoreKey
+
+	// The address that is capable of executing a MsgUpdateParams message.
```

Comment:
`MsgUpdateParams` is not defined on this module, no? Do you mean `UpdateLane` / `RegisterLane`